src/renderer/components/dock/dock.store.ts

```diff
@@ -89,6 +88,21 @@ export interface DockStorageState {
   isOpen?: boolean;
 }
 
+export interface DockTabChangeEvent {
+  tab: DockTab;
+  tabId: TabId;
+  prevTab?: DockTab;
+}
+
+export interface DockTabChangeEventOptions extends IReactionOptions {
+  tabKind?: TabKind; // filter: by dockStore.selectedTab.kind == tabKind
+  dockIsVisible?: boolean; // filter: dock and selected tab should be visible (default: true)
```

Comment:
Would be nice if those comments were doc comments instead.

Reply:
Updated